sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Count.java

```diff
@@ -101,6 +89,18 @@ private Count() {
    * PCollection<KV<String, Long>> wordCounts =
    *     words.apply(Count.<String>perElement());
    * } </pre>
+   */
+  public static <T> PTransform<PCollection<T>, PCollection<KV<T, Long>>> perElement() {
+    return new PerElement<>();
+  }
+
+  /**
+   * {@code Count.PerElement<T>} takes a {@code PCollection<T>} and returns a
```

Comment:
It seems reasonable to keep this block in the javadoc of perElement(), and reduce the javadoc of PerElement to simply `Implementation of {@link #perElement}.`